.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Tests
+on:
```

Comment:
can we add the workflow dispatch so can run test manually?

Reply:
Yep

Reply:
thanks